src/test/java/org/jpeek/MainTest.java

```diff
@@ -43,8 +44,8 @@
 public final class MainTest {
 
     @Test
-    public void createsXmlReports() throws IOException {
-        final Path output = Files.createTempDirectory("").resolve("x3");
+    public void createsXmlReports(@TempDir final Path temp) throws IOException {
+        final Path output = temp.resolve("x3");
```

Comment:
@dgarus Can't we just use output as a tempdir ? `@TempDir final Path output`

Reply:
We cannot do it because of the parameter field
`org.jpeek.Main#overwrite`.
Therefore we'll get the exception:
`java.lang.IllegalStateException: Overwrite disabled. Directory/file already exists: ...\Local\Temp\junit12420498039255830335`

Reply:
@dgarus Thanks! Ok for me. The most important thing is that it get deleted afterwards.